curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/record-collection.english.md

```diff
@@ -17,128 +18,146 @@ There are several rules for handling incomplete data:
 If <code>prop</code> is <code>"tracks"</code> but the album doesn't have a <code>"tracks"</code> property, create an empty array before adding the new value to the album's corresponding property.
 If <code>prop</code> is <code>"tracks"</code> and <code>value</code> isn't empty (<code>""</code>), push the <code>value</code> onto the end of the album's existing <code>tracks</code> array.
 If <code>value</code> is empty (<code>""</code>), delete the given <code>prop</code> property from the album.
-<strong>Hints</strong><br>Use <code>bracket notation</code> when <a href="/learn/javascript-algorithms-and-data-structures/basic-javascript/accessing-object-properties-with-variables" target="_blank">accessing object properties with variables</a>.
-The `push` array method will be helpful here. Check out our <a href="https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/manipulate-arrays-with-push" target="_blank">Manipulate Arrays With push()</a> challenge to review how it works.
+<strong>Hints</strong><br>
```

Comment:
```suggestion
<strong>Notes/Hints</strong><br>
```

Reply:
OK, I'll make the changes :smiley:

Reply:
You can simply click Commit suggestion and it will make the commit.

Reply:
I saw the option, but I tested the changes locally again to actually "see" them.